src/layers/core/scatterplot-layer/scatterplot-layer-vertex.glsl

```diff
@@ -32,16 +32,22 @@ uniform float radius;
 uniform float radiusMinPixels;
 uniform float radiusMaxPixels;
 uniform float renderPickingBuffer;
+uniform float drawOutline;
+uniform float strokeWidth;
 
 varying vec4 vColor;
+varying vec2 uv;
+varying float strokeWidthRatio;
 
 void main(void) {
   // Multiply out radius and clamp to limits
   float radiusPixels = clamp(
     project_scale(radius * instanceRadius),
     radiusMinPixels, radiusMaxPixels
-  );
+  ) + drawOutline * strokeWidth / 2.0;
```

Comment:
This addition... are we adding the outline outside of the radius? 
Is that what we want, and if so is the doc updated to clarify this?